zone/questmgr.cpp

```diff
@@ -870,11 +870,10 @@ void QuestManager::depop_withtimer(int npc_type) {
 
 void QuestManager::depopall(int npc_type) {
 	QuestManagerCurrentQuestVars();
-	if(owner && owner->IsNPC() && (npc_type > 0)) {
+	if ((owner && owner->IsNPC() && (npc_type > 0)) || npc_type > 0) {
```

Comment:
Can probably just remove the owner checks and QuestManagerCurrentQuestVars() altogether since they’re not used and don’t matter.

Reply:
Only check for NPC type > 0 basically.

Reply:
Fair enough, for some reason my mind was set on continuing to check for ownership and IsNPC but that is not relevant with this.